Cubical/Experiments/ZCohomologyOld/MayerVietorisUnreduced.agda

```diff
@@ -1,384 +0,0 @@
-{-# OPTIONS --safe #-}
```

Comment:
@aljungstrom   Should this be kept or deleted?

Reply:
This one does not check.

Reply:
Commented out whole file.

Reply:
@aljungstrom Can't we just delete this then?

Reply:
go ahead!